src/analyzer/protocol/ssl/tls-handshake-analyzer.pac

```diff
@@ -272,15 +272,48 @@ refine connection Handshake_Conn += {
 		return true;
 		%}
 
-	function proc_ec_server_key_exchange(rec: HandshakeRecord, curve_type: uint8, curve: uint16) : bool
+	function proc_ecdhe_server_key_exchange(rec: HandshakeRecord, curve_type: uint8, curve: uint16, point: bytestring, signed_params: bytestring) : bool
 		%{
 		if ( curve_type == NAMED_CURVE )
 			BifEvent::generate_ssl_server_curve(bro_analyzer(),
 			  bro_analyzer()->Conn(), curve);
+			BifEvent::generate_ssl_ecdh_server_params(bro_analyzer(),
```

Comment:
Are these and the following lines only supposed to be executed when it is a named curve, or can they be raised each time?

I assume they all only are appropriate in case a named curve is used - however this is not what happens at the moment :)